src/util/models.py

```diff
@@ -76,6 +76,34 @@ def remove_model(cr, model, drop_table=True, ignore_m2m=()):
 
     # remove references
     for ir in indirect_references(cr):
+        if ir.company_dependent_comodel:
+            if ir.company_dependent_comodel == "ir.model":
+                # clean on delete set null
+                cr.execute("SELECT id FROM ir_model WHERE name = %s", [model])
+                [mod_id] = cr.fetchone() or [None]
+                if mod_id:
+                    cr.execute(
+                        format_query(
+                            """
+                            UPDATE {table}
+                               SET {field} = (
+                                   SELECT jsonb_object_agg(
+                                          key,
+                                          CASE
+                                              WHEN value::int4 in %s THEN NULL
+                                              ELSE value::int4
+                                          END)
+                                     FROM jsonb_each_text({field})
+                                   )
+                             WHERE {field} IS NOT NULL
+                               AND {field} @? %s
+                            """,
+                            table=ir.table,
+                            field=ir.res_id
+                        ),
+                        [ids, "$.* ? (@ == {})".format(mod_id)]
```

Comment:
`ids` is not defined.
```suggestion
                        [mod_id, "$.* ? (@ == {})".format(mod_id)]
```